src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/CookieWarning.java

```diff
@@ -74,22 +74,31 @@ public static void checkCookie() {
 				return;
 			}
 			if (timeLine != null) {
-				String[] digits = timeLine.split(" ");
+				String[] digits = timeLine.replaceAll("(\u00a7.)", "").split(" ");
 				int minutes = 0;
 				try {
-					for (String digit : digits) {
-						if (digit.endsWith("y")) {
-							digit = digit.substring(0, digit.length() - 1);
-							minutes += Integer.parseInt(digit) * 525600;
-						} else if (digit.endsWith("d")) {
-							digit = digit.substring(0, digit.length() - 1);
-							minutes += Integer.parseInt(digit) * 1440;
-						} else if (digit.endsWith("h")) {
-							digit = digit.substring(0, digit.length() - 1);
-							minutes += Integer.parseInt(digit) * 60;
-						} else if (digit.endsWith("m")) {
-							digit = digit.substring(0, digit.length() - 1);
-							minutes += Integer.parseInt(digit);
+					for (int i = 0; i < digits.length; i++) {
+						if (i % 2 == 1) continue;
+
+						String number = digits[i];
+						String unit = digits[i + 1];
+						long val = Integer.parseInt(number);
+						switch (unit) {
+							case "Years":
+								minutes += val * 525600;
+								break;
+							case "Months":
+								minutes += val * 43200;
+								break;
+							case "Days":
+								minutes += val * 1440;
+								break;
+							case "Hours":
+								minutes += val * 60;
+								break;
+							case "Minutes":
+								minutes += val;
+								break;
```

Comment:
shouldn't this also include changes from https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/432/commits/99a0fe53a9d2f0973dd210d5f022642c992d9ad5 https://github.com/NotEnoughUpdates/NotEnoughUpdates/pull/433/commits/877c7f57f8cf4030fc25d996aad9a17e7d564cf6?

Reply:
perhaps